### ISSUE TITLE
Analytics Dashboard: Build a central Dashboard for a high-level overview of pagewise and XB performance metrics

### ISSUE THREAD

**Issue by PriyaBhatia0210** [humanmade/altis-analytics#119]
**User Story:**
As a Head of Marketing, I want to get a high-level overview of pages using personalized content and its relative page metrics.

**Requirement:**
A central location to see performance of my website and navigate to all the personalized content created by the editorial team.  

**Acceptance Criteria:**
Given: I am in the WP Admin
When: I select Analytics Dashboard from the left hand side bar
Then: A list of all web pages alongwith no. of XBs on each page should be displayed 
And: It should also display total no. of page views and the last modified date & time of that page

**UI Design:**
![image](https://user-images.githubusercontent.com/55112796/105905779-db3e1200-5ff0-11eb-85c4-788afcc96af3.png)

**Note:** 
1. This page is called 'Analytics Dashboard' and not XB Analytics Dashboard. 
2. Ignore the 'Page Bounce Rate' and 'Time on page' from this design.

**Comment by PriyaBhatia0210** [comment]
We are not including bounce rate and time on page on this list for the MVP. (Priya to update the description)
Also, remove the button from the AC

**Comment by noeltock** [comment]
I’ve taken a fresh approach to many of the things here and am generally resetting all assumptions.

The one thing that I’d to work against, is this idea that there’s an “analytics dashboard” that lives somewhere else, as a distinct thing, with a distinct name (or even abbreviation), that someone has to click into. I think this is fundamentally wrong. Why do DXP’s exist? Why personalise content? Why test headlines?

Bringing brands and audiences closer together is fundamental to what we do. It’s not “CMS + Analytics package”, we just experienced that with the "VIP + Parsely" deal, visionless. We're Altis, the DXP that's going to take WordPress out of its bubble — and we’re merely scratching the surface (later progressions include further ML augmentation, customer journey orchestration, etc.).

More tangibly and longer term, our unique differentiators are not measured by their potential to execute, but in their continued and frequent usage. Where WordPress stops and Altis begins, needs to be the lifeblood, the feedback loop, front and centre. That value-add needs to be on blast.

But.. but.. MVP? I hear you, but the skateboard here isn’t some tab at the bottom of the WP menu. It’s the new _discover weekly_ playlist you see when you open Spotify, it’s that _full-width movie trailer_ you haven’t watched yet autoplaying at the top of Netflix.  It’s all these apps you use daily, driving you to take action on new value, new creation, new experiences, whatever — that’s the skateboard. Let’s build that.

So we have to sift our data and find the opportunities, seeing what our customers can't yet — obligatory Nolan gif...

![anigif_sub-buzz-661-1576783493-4](https://user-images.githubusercontent.com/781166/108220572-ed225900-7147-11eb-9f1f-7c77890bba20.gif)

Ignoring WordPress UX/UI rules for a moment and having some fun in Sketch, I've doodled some ideas which I hope can contribute to a more ambitious conversation:

![Altis-Dashboard-v1](https://user-images.githubusercontent.com/781166/108220967-5609d100-7148-11eb-8df6-5410078c1ecb.jpg)

We have all this cool shit, XB's, AB headlines, audiences... but where's the user action? Let's bring them to life, let's put them front and centre. Let's go big, be proud of the tools we've built — lets make this stuff the first thing a user sees when they log in and get them to take action.
